### ISSUE TITLE
Headless not collecting screenshots

### ISSUE THREAD

**Issue by mariobrosch** [GoogleChrome/lighthouse#5815]
I'v installed lighthouse with NPM on my machine with the latest version (yesterday).

I run LH with the following script:
`lighthouse http://localhost:4200/ --output=json --output-path=./lighthouse-result.json --chrome-flags='--headless'`

The result is a JSON file with all the data, but the performance test fails.
The speed index has an error on the performance category with the following error: `Chrome didn't collect any screenshots during the page load. Please make sure there is content visible on the page, and then try re-running Lighthouse. (SPEEDINDEX_OF_ZERO)`

When I remove the headless flag the error disappears and I have a performance metric available. 

I expect the performance indicator is provided when using the headless flag.

**Comment by justinribeiro** [comment]
Hi! Could you provide some additional information, as well as potentially a URL that the issue occurs on?

#### Environment Information
* Affected Channels: 
* Chrome version:
* Lighthouse version:
* Node.js version:
* Operating System:

**Comment by mariobrosch** [comment]
Hi, I don't have my laptop with me, but really want to answer your question.

I have all the latest versions. 
Lighthouse 3.0.3
Windows 10 with all updates
Chrome latest stable (68)
Node updated to the latest also this week (8.11.3)

I'm trying a local website (local host) which is running fine. Other metrics are calculated, only this issue. 

A colleague with only an older windows (8.1) is running and gets a performance metric, but his machine isn't performing well

Hope you can help me out on this

**Comment by patrickhulce** [comment]
It's hard to debug without a URL to reproduce this issue :/ 

Is there a public page that you can hit this issue on reliably so we can take a look?

**Comment by mariobrosch** [comment]
I just tried google.com and it returns a performance metric. So no error. When I switch back to the localhost it gives back the same error :(.

**Comment by mariobrosch** [comment]
Tried a website on our local test environment and it returns a valid performance value. Tried http-server on my dist folder of my ember website and that also returns a questionmark (so the error I mentioned in the start of this thread). What can be possibly wrong on my machine that prevents on local websites to perform this?

**Comment by wardpeet** [comment]
could you try to save your artifacts with --save-assets maybe it can help us. Is it possible to send us a link to the source code? or create a replica?

**Comment by mariobrosch** [comment]
When I was trying to execute the --save-assets version of the command I got results back on my localhost. I thought 'this is strange, why?'. And I removed the extra parameter and ran it again. Still results :). 

I didn't change anything and for what I can see none of the packages or used software has been updated. I didnt record the chrome version exactly (just the major version), maybe there was a slight update. Anyways. The problem has been solved. Thank you for thinking with me.
